src/prog_models/models/pneumatic_valve.py

```diff
@@ -181,14 +181,13 @@ def gas_flow(self, pIn, pOut, C, A):
 
         if pIn/pOut>=threshold:
             return C*A*pIn*sqrt(k/Z/R/T*(2/(k+1))**((k+1)/(k-1)))
-        elif pIn/pOut<threshold and pIn>=pOut:
+        if pIn/pOut<threshold and pIn>=pOut:
```

Comment:
```suggestion
        elif pIn>=pOut:
```
The line before checks the first part of the conditional, so we can remove that and make it an elif